lib/asyn-thread.c

```diff
@@ -197,6 +204,16 @@ void destroy_thread_sync_data(struct thread_sync_data * tsd)
   if(tsd->res)
     Curl_freeaddrinfo(tsd->res);
 
+#ifdef HAVE_SOCKETPAIR
+  /* close socket pair */
+  if(tsd->sock_pair[0] != INVALID_SOCKET) {
+    close(tsd->sock_pair[0]);
```

Comment:
please use `sclose()` instead of `close()` to close sockets!

Reply:
done.